honeybee/face.py

```diff
@@ -548,14 +548,65 @@ def apertures_by_ratio_rectangle(self, ratio, aperture_height, sill_height,
             aperture._parent = self
             self._apertures.append(aperture)
 
+    def apertures_by_width_height_rectangle(self, aperture_height, aperture_width,
```

Comment:
Maybe this needs to have the word repeating somewhere. It is easy to miss the `s` in the call. I couldn't come up with a good short name and `repeating_apertures_by_width_height_rectangle` is too long. Just an observation.

Reply:
I'm wondering if it makes sense to use the word `repeating` instead of `rectangle`. Basically, I want to communicate that, just like `apertures_by_ratio_rectangle`, this `apertures_by_width_height_rectangle` method will fill any rectangular portions identified within the `Face3D` with repeating rectangular windows. Let me think about this since a naming change like this should happen to both of these methods and will require PRs to a few others that are making use of them. If we eventually decide that we want to change it, I'll send another round of PRs.